data-loader/data/v2/schema.graphql

```diff
@@ -74,6 +75,13 @@ type PropertyConfigurationItem {
   url: String @dgraph(pred: "url")
 }
 
+type Barcode {
+  id: ID!
+  gtin: String! @id @dgraph(pred: "code") @search(by: [exact, trigram])
```

Comment:
I like this using `code` in dgraph, kind of as our `id` look up.
Have created `interaction_group_id` and `interaction_id` in my other PR, but definitely complicates things a little bit...